app/depict/src/main/java/org/openscience/cdk/depict/FreeHepWrapper.java

```diff
@@ -128,7 +128,17 @@ public String toString() {
             result = result.replaceAll("\"([-+0-9.]+)px\"", "\"$1mm\"");
         }
         if (fmt.equals(Depiction.EPS_FMT)) {
-            String nl = System.getProperty("line.separator");
+            String nl;
+            // We should determine new-line separator (nl) not from OS type, but from the line-endings
+            // in the actual EPS outpu; there is nothing that would prevent us from generating Unix-style
+            // file in Windows :)
+            if( result.contains("\n\r")) {
```

Comment:
'+1' was about to say when reading the previous commit you might need this. Good to see it's here but should'n't it be *\r\n* not *\n\r*. As a general rule of thumb is good to read both but only write one (e.g. our Molfile writer should only ever use \n - whether it does actually do that of course is another mater).

Reply:
You are write, will fix right now...